.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -289,3 +289,50 @@ cross_RekognitionVideoDetection:
             sqs:
   services:
     rekognition:
+cross_DetectFaces:
```

Comment:
This is pretty similar to a couple of existing examples and should be part of one of them instead of a new example:
* rekognition_DetectFaces (single-service example)
* cross_RekognitionPhotoAnalyzer (cross-service example)

Reply:
Almost. I'll add it to cross_RekognitionPhotoAnalyzer and change the existing entry to make ses an additional service.